### DISCUSSION TITLE
Motion detection on timeline

### DISCUSSION THREAD

**Discussion by grembling22** [blakeblackshear/frigate#8655] [Ideas]
I used to have my config setup to record motion only. It was nice because on the recording timeline it would show things that didnt trigger a full detection event like deer in the back yard, or smaller birds or even people that didnt trigger that didnt meet the threshold to trigger a full person detection event (usually happens on dark nights). I then switched to 24/7 recording and lost the ability to show motion events. I was wondering if there was any way to implement the ability to still see these motion detections on some sort of timeline.

**Comment by NickM-27** [comment]
This will come in a future version as part of a recordings view revamp

**Reply by grembling22** [reply]
Thats great to hear!
